analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestClassShouldHaveTestMethodTest.cs

```diff
@@ -31,32 +31,43 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class TestClassShouldHaveTestMethodTest
     {
+        private readonly VerifierBuilder verifierBuilder = new VerifierBuilder<TestClassShouldHaveTestMethod>();
+
         [DataTestMethod]
         [DataRow("2.5.7.10213")]
         [DataRow(Constants.NuGetLatestVersion)]
         public void TestClassShouldHaveTestMethod_NUnit(string testFwkVersion) =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\TestClassShouldHaveTestMethod.NUnit.cs",
-                new TestClassShouldHaveTestMethod(),
-                NuGetMetadataReference.NUnit(testFwkVersion));
+            verifierBuilder
+                .AddPaths(@"TestClassShouldHaveTestMethod.NUnit.cs")
```

Comment:
The `@` pain is not needed anymore :)
```suggestion
                .AddPaths("TestClassShouldHaveTestMethod.NUnit.cs")
```

Reply:
Same below